responses/06_custom-workflow.md

```diff
@@ -4,13 +4,7 @@ Now that we've learned how to quickly set up CI, let's try a more realistic use
 
 Our fictional team has a custom workflow that goes beyond the template we've used so far. We would like the following features:
 
-- **test against multiple targets** so that we know if our supported operating systems and Node.js versions are working
-- **dedicated test job** so that we can separate out build from test details
-- **access to build artifacts** so that we can deploy them to a target environment
-- **branch protections** so that the `master` branch can't be deleted or inadvertently broken
-- **required reviews** so that any pull requests are double checked by teammates
-- **obvious approvals** so we can merge quickly and potentially automate merges and deployments
-
+![custom-workflow-list](https://user-images.githubusercontent.com/6351798/88590869-a8eb3200-d018-11ea-9967-6ca4fbdf831d.png)
```

Comment:
On this one, we lost the accessible text.

Reply:
Thanks for the review, @hectorsector! I added the text back in with thumbnail icons since we now reference them throughout the course responses. Let me know what you think. I can remove them all together if that's better.